src/SimpleWallet/SimpleWallet.cpp

```diff
@@ -119,388 +119,445 @@ namespace po = boost::program_options;
-const command_line::arg_descriptor<std::string> arg_log_file = {"log-file", "Set the log file location", ""};
-const command_line::arg_descriptor<uint32_t> arg_log_level = { "log-level", "Set the log verbosity level", INFO, true };
-const command_line::arg_descriptor<bool> arg_testnet = { "testnet", "Used to deploy test nets. The daemon must be launched with --testnet flag", false };
-const command_line::arg_descriptor<bool> arg_reset = { "reset", "Discard cache data and start synchronizing from scratch", false };
-const command_line::arg_descriptor< std::vector<std::string> > arg_command = { "command", "" };
-
-
-bool parseUrlAddress(const std::string& url, std::string& address, uint16_t& port) {
-  auto pos = url.find("://");
-  size_t addrStart = 0;
-
-  if (pos != std::string::npos) {
-    addrStart = pos + 3;
-  }
-
-  auto addrEnd = url.find(':', addrStart);
-
-  if (addrEnd != std::string::npos) {
-    auto portEnd = url.find('/', addrEnd);
-    port = Common::fromString<uint16_t>(url.substr(
-      addrEnd + 1, portEnd == std::string::npos ? std::string::npos : portEnd - addrEnd - 1));
-  } else {
-    addrEnd = url.find('/');
-    port = 80;
-  }
-
-  address = url.substr(addrStart, addrEnd - addrStart);
-  return true;
-}
-
-void seedFormater(std::string& seed){
-  const unsigned int word_width = 12;
-  const unsigned int seed_col = 5;
-  std::string word_buff;
-  std::vector<std::string> seed_array;
-  unsigned int word_n = 0;
-  for (unsigned int n = 0; n <= seed.length(); n++){
-    if (seed[n] != 0x20 && seed[n] != 0x0A && seed[n] != 0x00){
-      word_buff.push_back(seed[n]);
-    } else {
-      if (!word_buff.empty()){
-        seed_array.push_back(word_buff);
-        word_buff.clear();
-      }
-    }
-  }
-  seed.clear();
-  seed.append("\n ");
-  for (std::string word : seed_array){
-    seed.append(word);
-    for (unsigned int k = 2; k <= word_width - word.length() && word.length() <= word_width; k++) seed.append(" ");
-    seed.append(" ");
-    word_n++;
-    if (word_n >= seed_col){
-      word_n = 0;
-      seed.append("\n ");
-    }
-  }
-}
-
-inline std::string interpret_rpc_response(bool ok, const std::string& status) {
-  std::string err;
-  if (ok) {
-    if (status == CORE_RPC_STATUS_BUSY) {
-      err = "daemon is busy. Please try later";
-    } else if (status != CORE_RPC_STATUS_OK) {
-      err = status;
-    }
-  } else {
-    err = "possible lost connection to daemon";
-  }
-  return err;
-}
-
-template <typename IterT, typename ValueT = typename IterT::value_type>
-class ArgumentReader {
-public:
-
-  ArgumentReader(IterT begin, IterT end) :
-    m_begin(begin), m_end(end), m_cur(begin) {
-  }
+const command_line::arg_descriptor<std::string> arg_config_file = { "config-file",
+                                                                    "Specify configuration file",
+                                                                    "" };
```

Comment:
Adding everywhere `// clang-format on` and `// clang-format off` isn't a good idea.

"Format on save" is IDE feature and in big projects it should be turned off by default. So you would see only warnings from clang-format instead of using clang-format as a tool for automatated code editing.

Did you review the code while formating? Or did you just ran automatic formating tool and commited the code right after that?